chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
I think it should be a question of consistency, purely to avoid debates about which declarator should have been used, because either *can* makes sense, and at least in this case are readable (although I did move the declaration of `colId` further down in the commit I'm working on right now).